### PR TITLE
Add support for msvc with cmake version pre 3.9 (fixes #4429)

### PR DESCRIPTION
**Pull request by MocquillonCedric**
As explained in the bug ticket, I changed the macro to support cmake version before 3.9 on windows (multi target issue). 
It's my first PR in opensource world! So be free to tell me if I made something wrong.

### CODE REVIEW

**Comment by psi29a** [comment]
@AnyOldName3 Does this look correct to you?

**Comment by AnyOldName3** [comment]
LGTM

**Comment by psi29a** [comment]
OK, waiting on AV to do it's thing. Thanks. :)

**Comment by psi29a** [comment]
@MocquillonCedric : you mind add an entry to the Changelog for bug 4429?
https://github.com/akortunov/openmw/blob/master/CHANGELOG.md

**Comment by akortunov** [comment]
> you mind add an entry to the Changelog for bug 4429?

And you do not forget to update AUTHORS.md.

**Comment by psi29a** [comment]
They can update the AUTHORS.md if they want too. :)

File: AUTHORS.md [hunk 1]
```diff
@@ -37,6 +37,7 @@ Programmers
     Britt Mathis (galdor557)
     Capostrophic
     cc9cii
+	Cédric Mocquillon
```
Comment (akortunov) [inline AUTHORS.md]: Is this a tab? Usually we use 4 space characters instead.

Comment (psi29a) [reply AUTHORS.md]: Yeah, convert to 4 spaces then I'll merge this. Thank you! :)

Comment (MocquillonCedric) [reply AUTHORS.md]: Sorry for this, I just pushed the changes

**Comment by psi29a** [comment]
Thanks!

**Pull request merged**